exporter/gcloudpubsubexporter/README.md

```diff
@@ -0,0 +1,23 @@
+# Google Pubsub Exporter
+
+This exporter sends OTLP messages to a Google Cloud [Pubsub](https://cloud.google.com/pubsub) topic.
+
+The following configuration options are supported:
+
+* `project` (Required): The Google Cloud Project of the topics.
+* `validate_existence`(Optional): Checks the existence of the topic, but this requires admin permissions to validate 
```

Comment:
It would be good to list out the actual permission names required, both for normal use and when setting this flag.

Also I would consider removing this flag as it might just complicate things and failing when sending messages isn't that bad, many of our exporters work that way.

Reply:
This check is interesting. This verification could make sense to be implemented on exporter startup (how you implemented it), as this would have a fast feedback loop and prevent trying to send data that is going to fail. However, topics can be created at anytime. Since the collector uses a static config, would it be safe to assume that the topic name is known and has been created prior to configuring a collector? Or is it best to assume a fully asynchronous approach and never assume a topic exists?